ansible/roles/gpu/tasks/nvidia-gpu-Flatcar.yaml

```diff
@@ -201,13 +212,38 @@
     dest: "/opt/lib/x86_64-linux-gnu/libnvidia-container.so"
     state: link
 
+- name: copy ubuntu build Dockerfile to copy container-hook binary
+  copy:
+    src: "./Dockerfile.ubuntu"
+    dest: "/tmp/Dockerfile.ubuntu"
+
+- name: build the ubuntu image
+  command: docker build -f /tmp/Dockerfile.ubuntu --tag hook-build ./
+
+- name: copy hook from image
+  command: docker run -v /opt/bin:/opt/bin -w /opt/bin hook-build:latest cp /bin/nvidia-container-runtime-hook .
```

Comment:
Actually, won't this only copy the symlink? Don't you need `/usr/bin/nvidia-container-toolkit`, `/usr/bin/nvidia-container-runtime`, and `/usr/bin/nvidia-container-runtime-hook`?

Reply:
IIUC the default behavior of cp is to copy content, not links

Reply:
```
       -L, --follow-link[=false]      Always follow symbol link in SRC_PATH
```

Reply:
Weird I can change that for sure.

Reply:
Fixed